ws-android/Dockerfile

```diff
@@ -5,65 +5,180 @@ FROM cloud9/workspace
 # https://github.com/lukin0110/docker-android-build
 MAINTAINER Jeremy Ellis <keyfreemusic@gmail.com>
 
+# -------------------------- Startup Stuff -----------------------------------------
+# move files from the files folder to the docker container
 ADD ./files/workspace /home/ubuntu/workspace
 
-RUN chmod -R g+w /home/ubuntu/workspace && \
+# Give owner and ubuntu group: read,write,execute permissions, others read only
+# Change owner to ubuntu
+RUN chmod -R 774 /home/ubuntu/workspace && \
     chown -R ubuntu:ubuntu /home/ubuntu/workspace
     
+    
+
+# Setup a bash file to auto load environment variables when in cloud 9
+# Add to this file for every environment variable and Path used
+RUN touch /etc/profile.d/android.sh && \
+    chmod -R 775 /etc/profile.d/android.sh && \
+    chown -R ubuntu:ubuntu /etc/profile.d/android.sh && \
+    printf "#!/bin/bash\n\n" > /etc/profile.d/android.sh  && \
```

Comment:
There are a few programming reasons why the printf lines might be useful especially with upgrading to new versions. Present systems have version tags at several locations that need to be changed on every upgrade.